controls/SV-244540.rb

```diff
@@ -28,8 +23,9 @@
   tag cci: ['CCI-000366']
   tag nist: ['CM-6 b']
 
-  describe command('grep -i nullok /etc/pam.d/system-auth') do
+  pam_auth_files = input('pam_auth_files')
```

Comment:
Since we unify the checks of all the pam files for this same setting this control, 244541 and their brother that checks smart cards will all be the same test.